linux_os/guide/system/permissions/partitions/mount_option_boot_nosuid/rule.yml

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
```

Comment:
What about RHEL 7, maybe RHEL 6 too?

Reply:
I'm working based on the RHEL 8 kickstart by @stevegrubb. Not sure to what extend this is applicable to older versions. We can always add them later en masse.